modules/date.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable quotes */
+import { DateTime } from "./luxon.js";
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Kindly remove this eslint disabling and fix the errors. It is important to maintain the linters as they are since they help write standardized and maintainable code.